closures/index.js

```diff
@@ -0,0 +1,77 @@
+//Q -Create a function to calculate the factorial of a number using closure
+function factorial(n) {
+  let result = 1;
+  for (let i = 1; i <= n; i++) {
+    (() => {
+      result = result * i;
+    })();
+  }
```

Comment:
Why iffe is needed?

Reply:
IIFE was not necessarily needed in this task, but as it was asked in asmt that use closures so i used iife function (normal func. can also be used ). which will use the result variable from its parents scope by the help of closures.

Reply:
Don't use unnecessary complicated concepts....always make your code simple and easy to read

Reply:
Surely, sir >